athletics.lic

```diff
@@ -2,22 +2,26 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#athletics
 =end
 
-custom_require.call(%w[common common-travel events drinfomon])
+custom_require.call(%w[common common-items common-travel events drinfomon])
 
 class Athletics
   include DRC
   include DRCT
+  include DRCI
 
   def initialize
     @settings = get_settings
     @athletics_options = get_data('athletics').athletics_options
 
     @climbing_target = get_data('athletics').practice_options[@settings.climbing_target]
+    have_climbing_rope = get_settings.have_climbing_rope
 
     start_exp = DRSkill.getxp('Athletics')
     @end_exp = [start_exp + 15, 29].min
 
-    if @climbing_target
+    if have_climbing_rope
+      climb_practice(get_settings.safe_room, 'climbing rope', false)
```

Comment:
I'm realizing that it might be easier to have the rope setup/usage/cleanup all in its own methods rather than trying to piggy-back in `climb_practice`